modules/core-functions/pages/base.adoc

```diff
@@ -16,6 +16,8 @@ Initiate a connection object.
 protocol (http:// or https://). If `certPath` is `None` and the protocol is https,
 a self-signed certificate will be used.
 * `graphname`: The default graph for running queries.
+* `gsqlSecret`: The secret key for GSQL. Created in AdminPortal of GraphStudio for the graph you wish to use.
```

Comment:
Added a cross-reference for users to learn about what a secret is, since secrets aren't just created in Admin Portal. Also, Admin Portal isn't considered by the application team to be part of GraphStudio. I think in future designs they will make the distinction more pronounced. 

Replace the description for `gsqlSecret` with the following:

> https://docs.tigergraph.com/tigergraph-server/current/user-access/managing-credentials#_secrets[Secret] used to authenticate requests.